### ISSUE TITLE
Best way to upgrade previous version

### ISSUE THREAD

**Issue by mattlokk** [timhul/ClassicSim#126]
Can I just copy the .exe into the old directory? are other files required?

**Comment by veluria** [comment]
Yes, the other files are required too. You can either copy everything into your previous version and choose to overwrite files (your profiles are safe) or you can just copy your profiles into the new folder.

**Comment by timhul** [comment]
You can sometimes copy your Saves/ folder, which I assume is what you actually want to store between versions. If the application does not but, then something has likely corrupted it, but that should rarely happen very often anymore.

If you have other local changes such as rotation, new items, etc. you should be able to copy all those directories as well.
